packages/renderer/src/lib/preferences/PreferencesConnectionDetailsLogs.spec.ts

```diff
@@ -45,10 +43,30 @@ const containerConnection: ProviderContainerConnectionInfo = {
   type: 'podman',
 };
 
-beforeAll(() => {
-  (window as any).getConfigurationValue = vi.fn();
-  (window as any).startReceiveLogs = vi.fn();
+beforeAll(async () => {
+  vi.stubGlobal('window', {
+    getConfigurationValue: vi.fn(),
```

Comment:
window mockups are generated by default now so I don't think we need this

Reply:
hmm, didn't know that, thx for pointing this. removed the stub and re-checked, tests are passing without them